data/mn/retired/Anne-Neu-a93779a2-69bc-4d3c-a8c6-530f3eccc341.yml

```diff
@@ -8,13 +8,13 @@ image: https://www.house.leg.state.mn.us/hinfo/memberimgls92/32B.gif?v=112221
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-02'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mn/government
```

Comment:
@bfossen-ce this will need to be moved into legislature.

Reply:
Updated